Externals/LuaJIT-proj/luajit.cmake

```diff
@@ -159,11 +159,6 @@ elseif ("${TARGET_TESTARCH}" MATCHES "LJ_TARGET_ARM")
 	set(TARGET_LJARCH "arm")
 elseif ("${TARGET_TESTARCH}" MATCHES "LJ_TARGET_PPC")
 	set(TARGET_LJARCH "ppc")
-	if ("${TARGET_TESTARCH}" MATCHES "LJ_LE 1")
-		set(TARGET_ARCH "-DLJ_ARCH_ENDIAN=LUAJIT_LE")
-	else()
-		set(TARGET_ARCH "-DLJ_ARCH_ENDIAN=LUAJIT_BE")
-	endif()
```

Comment:
Why this was removed, though?

Reply:
It's redundant as it's configured in the LuaJIT file from what I remember. Those lines don't do anything.